src/Polly.nuspec

```diff
@@ -141,16 +145,13 @@
       <group targetFramework="MonoTouch" />
       <group targetFramework="Xamarin.iOS" />
       <group targetFramework="portable-net45+win8+wpa81+wp8" />
-      <group targetFramework="dotnet">
-        <dependency id="System.Reflection" version="4.0.0" />
-        <dependency id="System.Collections" version="4.0.0" />
-        <dependency id="System.Diagnostics.Debug" version="4.0.0" />
-        <dependency id="System.Linq" version="4.0.0" />
-        <dependency id="System.Runtime" version="4.0.0" />
-        <dependency id="System.Runtime.Extensions" version="4.0.0" />
-        <dependency id="System.Threading" version="4.0.0" />
-        <dependency id="System.Threading.Tasks" version="4.0.0" />
+      <group targetFramework="netstandard1.0">
+        <dependency id="System.Diagnostics.Debug" version="4.0.11" />
```

Comment:
Instead of listing the specific assemblies, the recommendation is to reference .NET Standard instead of the individual dependencies (see https://docs.microsoft.com/en-us/dotnet/articles/standard/library)